nodedb.go

```diff
@@ -243,10 +251,13 @@ func (ndb *nodeDB) SaveFastNodeNoCache(node *fastnode.Node) error {
 	return ndb.saveFastNodeUnlocked(node, false)
 }
 
-// setFastStorageVersionToBatch sets storage version to fast where the version is
+// SetFastStorageVersionToBatch sets storage version to fast where the version is
 // 1.1.0-<version of the current live state>. Returns error if storage version is incorrect or on
 // db error, nil otherwise. Requires changes to be committed after to be persisted.
-func (ndb *nodeDB) setFastStorageVersionToBatch(latestVersion int64) error {
+func (ndb *nodeDB) SetFastStorageVersionToBatch(latestVersion int64) error {
```

Comment:
why is this now exported?

Reply:
all APIs which are used outside of nodeDB is exported, the function export rules are messy in the current imp, but just following the current imp